### DISCUSSION TITLE
Without installer credentials will this plugin work

### DISCUSSION THREAD

**Discussion by pentium10** [wlcrs/huawei_solar#304] [General]
Without installer credentials will this plugin work?
Is there a way for the installer to create a separate account just for the purpose of this integration?

**Comment by wlcrs** [comment]
Yes. Just leave the checkbox 'Elevated permissions' unchecked in the installation wizard.

**Comment by pentium10** [comment]
so credentials still needed, or when you leave unchecked, no credentials at all are needed?
